src/avatar.jsx

```diff
@@ -1,8 +1,45 @@
 import React from 'react';
-import StylePropable from './mixins/style-propable';
 import Colors from './styles/colors';
 import getMuiTheme from './styles/getMuiTheme';
 
+function getStyles(props, state) {
+  const {
+    backgroundColor,
+    color,
+    size,
+    src,
+  } = props;
+
+  const {
+    avatar,
+  } = state.muiTheme;
+
+  const styles = {
+    root: Object.assign({
```

Comment:
This is a ternary vampire, sucking the life out of readability :laughing: :laughing: 

```js
root: {
  color,
  backgroundColor,
  userSelect: 'none',
  display: 'inline-block',
  textAlign: 'center',
  lineHeight: `${size}px`,
  fontSize: size / 2 + 4,
  borderRadius: '50%',
  border: src ? `solid 1px ${avatar.borderColor}` : undefined,
  height: src ? size - 2 : size,
  width: src ? size - 2 : size,
},
```